examples/provide-liquidity/components/wallet/Wallet.tsx

```diff
@@ -107,6 +135,29 @@ export const WalletSection = () => {
     />
   );
 
+  const Balance = () => {
+    return (
+      <HStack w="256px" py={2} fontSize="13px" justify="space-between">
+        <Flex gap={1}>
+          <Text fontWeight="semibold">
+            Balance: {osmoBalance?.amount || '--'}
```

Comment:
`.amount` == `0` will set balance to `--`. Change to ?? coalescing nullish.

Reply:
ok ,  {osmoBalance?.amount ?? '--'}